Add readme (#3)
Gibe/Umbraco.Community.ModelsBuilder.PropertyOverride: .github/README.md

# Umbraco.Community.ModelsBuilder.PropertyOverride 

[![Downloads](https://img.shields.io/nuget/dt/Umbraco.Community.ModelsBuilder.PropertyOverride?color=cc9900)](https://www.nuget.org/packages/Umbraco.Community.ModelsBuilder.PropertyOverride/)
[![NuGet](https://img.shields.io/nuget/vpre/Umbraco.Community.ModelsBuilder.PropertyOverride?color=0273B3)](https://www.nuget.org/packages/Umbraco.Community.ModelsBuilder.PropertyOverride)
[![GitHub license](https://img.shields.io/github/license/Gibe/Umbraco.Community.ModelsBuilder.PropertyOverride?color=8AB803)](../LICENSE)

> This package provides functionality for overriding Models Builder property implementations.

Reviewer: ```suggestion
This package re-introduces functionality for overriding Models Builder property implementations that was lost in v9+.
```

[![NuGet](https://img.shields.io/nuget/vpre/Umbraco.Community.ModelsBuilder.PropertyOverride?color=0273B3)](https://www.nuget.org/packages/Umbraco.Community.ModelsBuilder.PropertyOverride)
[![GitHub license](https://img.shields.io/github/license/Gibe/Umbraco.Community.ModelsBuilder.PropertyOverride?color=8AB803)](../LICENSE)

This package provides functionality for overriding Models Builder property implementations.

Umbraco versions 13.x - 15.x are supported.

Properties can be marked up with a re-implemented [ImplementPropertyType("alias")] attribute, which causes them to be ignored when generating models through the default Models Builder.

In addition, classes can be marked up with [ImplementAllPropertyTypes], which causes all properties on the class to be ignored, this is functionally equivalent to adding [ImplementPropertyType("alias")] to every property on the class.

> ## Examples

Reviewer: Maybe add in a link to the old docs?

[ImplementPropertyType("alias")] to ignore properties on an individual basis:

> <img alt="ImplementPropertyType" src="https://github.com/Gibe/Umbraco.Community.ModelsBuilder.PropertyOverride/blob/develop/docs/screenshots/implement-property-type.png">

Reviewer: ```suggestion
![ImplementPropertyType](docs/screenshots/implement-property-type.png)
```

[ImplementAllPropertyTypes] to ignore all properties:

> <img alt="ImplementAllPropertyTypes" src="https://github.com/Gibe/Umbraco.Community.ModelsBuilder.PropertyOverride/blob/develop/docs/screenshots/implement-all-property-types.png">

Reviewer: ```suggestion
![ImplementAllPropertyTypes](docs/screenshots/implement-all-property-types.png)
```
